tests/Feature/OrderCancellation/OrganisationWithTaxTest.php

```diff
@@ -274,14 +249,8 @@ public function cancels_and_refunds_order_with_multiple_tickets_with_tax_and_fix
         $attendeeIds = $attendees->pluck('id')->toArray();
         $response = $this->actingAs($this->getAccountUser())
             ->post("event/order/$order->id/cancel", [
-                'attendees' => [ $attendeeIds[0] ],
+                'attendees' => [$attendeeIds[0]],
```

Comment:
Sorry to be pedantic again. This change is breaking the test logic. In this test we are testing multiple attendees getting cancelled so please swap out the single attendee for the multiple like below:

```
                'attendees' => [
                    $attendeeIds[0],
                    $attendeeIds[1],
                ],
```